rest/tsig_key.go

```diff
@@ -136,6 +138,13 @@ func (s *TsigService) Delete(name string) (*http.Response, error) {
 }
 
 var (
+	// Status Codes used
+	BadRequestCode          = 400
+	NotFoundStatusCode      = 404
+	AlreadyExistsStatusCode = 409
```

Comment:
Please use the existing constants in stdlib, they are in the http package. e.g. `http.StatusNotFound`